llm/ollama/README.md

```diff
@@ -0,0 +1,296 @@
+# Ollama: Run quantized LLMs on CPUs and GPUs
+<p align="center">
+  <img src="https://i.imgur.com/HfqnGVA.png" width="400">
+</p>
+
+[Ollama](https://github.com/ollama/ollama) is popular library for running LLMs on both CPUs and GPUs. 
+It supports a wide range of models, including quantized versions of `llama2`, `llama2:70b`, `mistral`, `phi`, `gemma:7b` and many [more](https://ollama.com/library). 
+You can use SkyPilot to run these models on CPU instances on any cloud provider, Kubernetes cluster, or even on your local machine. 
+
```

Comment:
Shall we add a quick pointer here something like: (Of course, GPUs are supported too if you have them.)

Should we add a quick section to the end of this doc on how to enable GPUs? Reason is a reader currently sees only CPU mentioned in this doc.

Reply:
Good call - added pointers for GPUs